druid/src/widget/image.rs

```diff
@@ -106,7 +154,14 @@ impl<T: Data> Widget<T> for Image {
     }
 }
 
-/// Stored Image data.
+/// Loaded image data.
```

Comment:
How about *Processed image data.*

Reply:
definitely, i struggled with this a bit. i was hoping to get a correction for this one, frankly.